src/GraphScene.tsx

```diff
@@ -159,6 +160,11 @@ export interface GraphSceneProps {
    */
   renderNode?: NodeRenderer;
 
+  /**
+   * Render a custom cluster label
+   */
+  renderClusterLabel?: ClusterLabelRenderer;
```

Comment:
We should just make this render the whole cluster and let the user override whatever they want imo

Reply:
updated